testassets/InteropTestsClient/InteropTestsClient.csproj

```diff
@@ -44,13 +44,8 @@
     <PackageReference Include="System.Net.Http.WinHttpHandler" Version="$(SystemNetHttpWinHttpHandlerPackageVersion)" />
   </ItemGroup>
 
-  <ItemGroup Condition="$([MSBuild]::VersionGreaterThanOrEquals($(TargetFrameworkVersion), 5.0))">
+  <ItemGroup Condition="'$(TargetFramework)' == 'net5.0' OR '$(TargetFramework)' == 'net6.0' OR '$(TargetFramework)' == 'net7.0'">
```

Comment:
Any reason we're no longer `VersionGreaterThanOrEquals`? It's a bit more future-proof.

Reply:
I couldn't get it to work after adding net7.0 target 🤷